### PR TITLE
add back missing tutorials; fix links

### PR DESCRIPTION
**Pull request by aaronmarkham**
## Description ##
Fixing up some missing tutorials and/or broken links.
I added back some things that seemed missing, but they might have just been renamed/relocated. I figured it would be better to fix the links/content now, then dedupe if that comes up.

### CODE REVIEW

File: docs/python_docs/python/tutorials/packages/gluon/custom_layer.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
```
Comment (ThomasDelteil) [inline docs/python_docs/python/tutorials/packages/gluon/custom_layer.md]: this tutorial is already there: https://mxnet.apache.org/api/python/docs/tutorials/extend/custom_layer.html

File: docs/python_docs/python/tutorials/packages/gluon/hybrid.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
```
Comment (ThomasDelteil) [inline docs/python_docs/python/tutorials/packages/gluon/hybrid.md]: This tutorial is already there: https://mxnet.apache.org/api/python/docs/tutorials/packages/gluon/blocks/hybridize.html

File: docs/python_docs/python/tutorials/packages/gluon/learning_rate_finder.md [hunk 3]
```diff
@@ -0,0 +1,332 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
```
Comment (ThomasDelteil) [inline docs/python_docs/python/tutorials/packages/gluon/learning_rate_finder.md]: already there https://mxnet.apache.org/api/python/docs/tutorials/packages/gluon/training/learning_rates/learning_rate_finder.html

File: docs/python_docs/python/tutorials/packages/gluon/learning_rate_schedules.md [hunk 4]
```diff
@@ -0,0 +1,345 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
```
Comment (ThomasDelteil) [inline docs/python_docs/python/tutorials/packages/gluon/learning_rate_schedules.md]: already there: https://mxnet.apache.org/api/python/docs/tutorials/packages/gluon/training/learning_rates/learning_rate_schedules.html

File: docs/python_docs/python/tutorials/packages/gluon/transforms.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
```
Comment (ThomasDelteil) [inline docs/python_docs/python/tutorials/packages/gluon/transforms.md]: All the links in this tutorial need to be updated, I'll take care of it in #16368

File: docs/python_docs/python/tutorials/packages/symbol/data.md [hunk 6]
```diff
@@ -0,0 +1,474 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
```
Comment (ThomasDelteil) [inline docs/python_docs/python/tutorials/packages/symbol/data.md]: I think we should get rid of Symbol tutorials

File: docs/python_docs/python/tutorials/packages/symbol/symbol.md [hunk 7]
```diff
@@ -0,0 +1,446 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
```
Comment (ThomasDelteil) [inline docs/python_docs/python/tutorials/packages/symbol/symbol.md]: let's get rid of symbol tutorials

File: docs/python_docs/python/tutorials/performance/backend/mkldnn/index.rst [hunk 8]
```diff
@@ -32,9 +32,15 @@ Intel MKL-DNN
 
       How to perform quantization with MKLDNN
 
+   .. card::
+      :title: Connectionist Temporal Classification
+      :link: speech_recognition/ctc.html
```
Comment (ThomasDelteil) [inline docs/python_docs/python/tutorials/performance/backend/mkldnn/index.rst]: that has nothing to do with mkldnn

File: docs/python_docs/python/tutorials/packages/gluon/hybrid.md (hunk 2 above)
Comment (aaronmarkham) [reply docs/python_docs/python/tutorials/packages/gluon/hybrid.md]: They're similarly named but the content seems quite different to me. Maybe same topic, but different...

File: docs/python_docs/python/tutorials/packages/gluon/index.rst [hunk 9]
```diff
@@ -25,7 +25,7 @@ Getting started
 
    .. card::
       :title: A 60-minute Gluon crash course
-      :link: ../../getting-started/crash-course/index.html
+      :link: ../../crash-course/index.html
```
Comment (aaronmarkham) [inline docs/python_docs/python/tutorials/packages/gluon/index.rst]:
```suggestion
      :link: ../../getting-started/crash-course/index.html
```

File: docs/python_docs/python/tutorials/packages/gluon/index.rst [hunk 10]
```diff
@@ -90,7 +90,7 @@ Data
 
    .. card::
       :title: Gluon Datasets and DataLoader
-      :link: data/datasets.html
+      :link: datasets.html
```
Comment (aaronmarkham) [inline docs/python_docs/python/tutorials/packages/gluon/index.rst]:
```suggestion
      :link: data/datasets.html
```

File: docs/python_docs/python/tutorials/packages/gluon/index.rst [hunk 11]
```diff
@@ -78,7 +78,7 @@ Data
 
    .. card::
       :title: Image Augmentation
-      :link: image-augmentation.html
+      :link: image_augmentation.html
```
Comment (aaronmarkham) [inline docs/python_docs/python/tutorials/packages/gluon/index.rst]:
```suggestion
      :link: image/image-augmentation.html
```

File: docs/python_docs/python/tutorials/packages/gluon/index.rst [hunk 12]
```diff
@@ -184,39 +216,57 @@ Advanced Topics
 
    .. card::
       :title: Custom Layers
-      :link: custom-layer.html
+      :link: custom_layer.html
```
Comment (aaronmarkham) [inline docs/python_docs/python/tutorials/packages/gluon/index.rst]:
```suggestion
      :link: blocks/custom-layer.html
```

File: docs/python_docs/python/tutorials/packages/gluon/index.rst [hunk 13]
```diff
@@ -184,39 +216,57 @@ Advanced Topics
...
 
       A guide to implementing custom layers.
 
    .. card::
       :title: Custom Operators
-      :link: https://mxnet.apache.org/versions/master/tutorials/gluon/customop.html
+      :link: customop.html
```
Comment (aaronmarkham) [inline docs/python_docs/python/tutorials/packages/gluon/index.rst]:
```suggestion
      :link: ../../extend/customop.html
```

File: docs/python_docs/python/tutorials/packages/gluon/index.rst [hunk 14]
```diff
@@ -184,39 +216,57 @@ Advanced Topics
...
 
       Building custom operators with numpy.
 
    .. card::
       :title: Custom Loss
-      :link: custom-loss/custom-loss.html
+      :link: custom_loss/custom_loss.html
```
Comment (aaronmarkham) [inline docs/python_docs/python/tutorials/packages/gluon/index.rst]:
```suggestion
      :link: loss/custom-loss.html
```

File: docs/python_docs/python/tutorials/packages/gluon/index.rst [hunk 15]
```diff
@@ -125,7 +131,7 @@ Training
 
    .. card::
       :title: Loss Functions
-      :link: loss/loss.html
+      :link: loss.html
```
Comment (aaronmarkham) [inline docs/python_docs/python/tutorials/packages/gluon/index.rst]:
```suggestion
      :link: loss/loss.html
```

File: docs/python_docs/python/tutorials/packages/gluon/index.rst [hunk 16]
```diff
@@ -184,39 +216,57 @@ Advanced Topics
...
       :title: Custom Loss
-      :link: custom-loss/custom-loss.html
+      :link: loss/custom-loss.html
 
       A guide to implementing custom losses.
 
    .. card::
       :title: Gotchas using NumPy in Apache MXNet
-      :link: https://mxnet.apache.org/versions/master/tutorials/gluon/gotchas_numpy_in_mxnet.html
+      :link: gotchas_numpy_in_mxnet.html
```
Comment (aaronmarkham) [inline docs/python_docs/python/tutorials/packages/gluon/index.rst]:
```suggestion
      :link: ../ndarray/gotchas_numpy_in_mxnet.html
```

File: docs/python_docs/python/tutorials/packages/gluon/hybrid.md (hunk 2 above)
Comment (ThomasDelteil) [reply docs/python_docs/python/tutorials/packages/gluon/hybrid.md]: All the links of this one are outdated, I'd say let's remove it

File: docs/python_docs/python/tutorials/packages/symbol/data.md (hunk 6 above)
Comment (aaronmarkham) [reply docs/python_docs/python/tutorials/packages/symbol/data.md]: I don't disagree. Let's do that separately though. Other people might still be using that.

File: docs/python_docs/python/tutorials/performance/backend/mkldnn/index.rst (hunk 8 above)
Comment (aaronmarkham) [reply docs/python_docs/python/tutorials/performance/backend/mkldnn/index.rst]: That's a weird merge artifact... I'll remove it.

**Comment by aaronmarkham** [comment]
Closing this, so I can apply my updates after Soji's new organization is applied. (Plus I have to simplify the commit history.)

**Pull request closed without merging**